pkg/ext-proc/backend/vllm/metrics.go

```diff
@@ -107,9 +111,36 @@ func promToPodMetrics(metricFamilies map[string]*dto.MetricFamily, existing *bac
 		multierr.Append(errs, fmt.Errorf("metric family %q not found", ActiveLoRAAdaptersMetricName))
 	}
 
+	if loraMetrics != nil {
+		updated.CachedModels = make(map[string]int)
+		for _, label := range loraMetrics.GetLabel() {
+			if label.GetName() == "running_lora_adapters" {
+				if label.GetValue() != "" {
+					adapterList := strings.Split(label.GetValue(), ",")
+					for _, adapter := range adapterList {
+						updated.CachedModels[adapter] = 0
+					}
+				}
+			}
+		}
+
+	}
+
 	return updated, errs
 }
 
+func getLatestLoraMetric(loraRequests *dto.MetricFamily) (*dto.Metric, time.Time, error) {
```

Comment:
can you document why this metric is read this way? i.e., the value of the metric is the timestamp while the labels are the lora adapters.